deps/src/main/java/com/microsoft/azure/sdk/iot/deps/serializer/TwinProperty.java

```diff
@@ -58,13 +58,14 @@ protected void enableMetadata()
         this.reportMetadata = true;
     }
 
+    @SuppressWarnings("SameParameterValue") // Value for "propertyVersion" is currently null, but this method can be used generically as well.
```

Comment:
If propertyVersion is always null, wouldn't it better to change the signature here instead of suppression?

Reply:
we want this method to be generic, so no